domain/robstoll-lib/atrium-domain-robstoll-lib-common/src/main/kotlin/ch/tutteli/atrium/domain/robstoll/lib/creating/iterableAssertions.kt

```diff
@@ -71,15 +69,22 @@ fun <E : Any, T : Iterable<E?>> _iterableAll(
 }
 
 fun <E : Any> _hasNext(expect: Expect<Iterable<E>>): Assertion =
-    ExpectImpl.builder.createDescriptive(expect,
-        DescriptionBasic.HAS, RawString.create(DescriptionIterableAssertion.NEXT_ELEMENT)) { it.iterator().hasNext() }
+    ExpectImpl.builder.createDescriptive(
+        expect,
+        DescriptionBasic.HAS, RawString.create(NEXT_ELEMENT)
+    ) { it.iterator().hasNext() }
 
 fun <E : Any> _hasNotNext(expect: Expect<Iterable<E>>): Assertion =
-    ExpectImpl.builder.createDescriptive(expect,
-        DescriptionBasic.HAS_NOT, RawString.create(DescriptionIterableAssertion.NEXT_ELEMENT)) {
+    ExpectImpl.builder.createDescriptive(
+        expect,
+        DescriptionBasic.HAS_NOT, RawString.create(NEXT_ELEMENT)
+    ) {
         !it.iterator().hasNext()
     }
 
+fun <E : Comparable<E>, T : Iterable<E>> _min(assertionContainer: Expect<T>): ExtractedFeaturePostStep<T, E?> =
+    ExpectImpl.feature.manualFeature(assertionContainer, MIN) { min() }
```

Comment:
I changed my mind as I did not think about that the return type of `min()` is `T?` and not `T`. Use `ExpectImpl.feature.extractor` instead and check if the Iterable `it.iterator().hasNext()` => I updated the description of the task please see there.

Reply:
done